packages/vm/src/modules/file_system_cache.rs

```diff
@@ -96,22 +90,18 @@ impl FileSystemCache {
     }
 
     /// Stores a serialized module to the file system. Returns the size of the serialized module.
-    /// The serialized module size is a good approximation (~100.06 %) of the in-memory module size.
-    /// It should not be considered as the exact in-memory module size.
+    /// Uses loupe to get a value for the module size (artifact + store) in bytes.
     pub fn store(&mut self, checksum: &Checksum, module: &Module) -> VmResult<usize> {
```

Comment:
Now that we do not care amout the size on disk anymore, I'd remove the `module_size` from here and return `VmResult<()>` instead. The caller can then get the size via the `Module`.

Same for `store` above

Reply:
Makes sense.

Reply:
`Module` is part of wasmer, so we cannot modify it.

In the future, it would be nice if it provides a `size()` helper that gives the size, and hides the implementation.